docs/comparison.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This document explains how to run an automated load-test comparison.
+This document explains how to run an automated load-test comparison. This configuration is aimed to experienced users running recurrent automatic comparisons; if you want to run a manual comparison once or twice, you should run two individual load-tests and generate a comparison report. See [Comparing load test reports](compare.md) for more information.
```

Comment:
Recurrent or recurring? I'm not sure :thinking:

Reply:
Merriam Webster does not know either :shrug:  https://www.merriam-webster.com/dictionary/recurrent#note-1

Reply:
> maybe we want to keep it in bold to

Which part? The whole thing seems too much. Something like this?
```suggestion
This document explains how to run an automated load-test comparison. **This configuration is aimed to experienced users** running recurring automatic comparisons; if you want to run a manual comparison once or twice, you should run two individual load-tests and generate a comparison report. See [Comparing load test reports](compare.md) for more information.
```

Reply:
Yep, looks good!